app/views/events/show.rb

```diff
@@ -2,7 +2,7 @@
 module Events
   class Show < ApplicationView
     def event_title
-      presenter.title
+      [presenter.title, t('site.events.list.page-title')]
```

Comment:
The return value of this method is used for the *on-page* title of the event. This needs to be a single string, i.e. the return value of `presenter.title`.